.github/workflows/spark-alchemy-workflow.yml

```diff
@@ -27,6 +27,11 @@ jobs:
         ports:
           - 5432:5432
     steps:
+      - uses: coursier/setup-action@v1
+        with:
+          jvm: zulu:8
+          apps: sbt
+
```

Comment:
This PR isn't being picked up by GitHub Actions to run. This is because the [triggers]() need to be updated to the following:

```yaml
on:
  push:
    branches:
      - '**'
      - '!gh-pages'
```

`branches` needs to be double star `**`, otherwise branch names that include slashes won't be picked up.